imports/plugins/included/taxes-avalara/client/accounts/exemption.js

```diff
@@ -6,11 +6,16 @@ import { Packages, Accounts } from "/lib/collections";
 import { Accounts as AccountsSchema } from "/lib/collections/schemas/accounts";
 import { TaxEntityCodes } from "/client/collections";
 
+let entityCodeList = [];
+let currentAccount;
+
 Template.taxSettingsPanel.helpers({
   account() {
     const sub = Meteor.subscribe("UserAccount", this.member.userId);
```

Comment:
**Update:**
I was wrong. The subscription can be moved into `onCreated` and it worked as expected (like you said).

```
Template.taxSettingsPanel.onCreated(function () {
  this.subscribe("UserAccount", Meteor.userId());
});

Template.taxSettingsPanel.helpers({
  account() {
    if (Template.instance().subscriptionsReady()) {
      console.log('sub ready');
      const account = Accounts.findOne({ _id: this.member.userId });
      console.log({ account });
      currentAccount = account;
      return account;
    }

    return null;
  },
........
```

I'll modify it.